test/unit.test.ts

```diff
@@ -0,0 +1,20 @@
+import { generateRandomSlugs } from "../slug";
```

Comment:
The best practice is to use the same name as the thing you are testing in the unit test file name, suffixed by .test. Please rename this one :)

Reply:
done